ui_framework/components/button/create_button_link.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import { KuiButtonLink } from './kui_button_link';
+import { CreateIcon } from '../icon/create_icon';
+
+export function CreateButtonLink(props) {
+  return <KuiButtonLink className="kuiButton--primary" {...props}>
```

Comment:
If `props` contains `className` it will override the existing `className` on this. This should probably be:

```js
const { className, ...rest } = props
const classes = classnames('kuiButton--primary', className)
return <KuiButtonLink className={classes} {...rest}>
```

or something like that.